src/sentry/tasks/post_process.py

```diff
@@ -55,6 +56,22 @@ def post_process_group(event, is_new, is_regression, is_sample, is_new_group_env
     """
     Fires post processing hooks for a group.
     """
+    with redis.clusters.get('default').map() as client:
```

Comment:
This can go wherever, open to changing it

Reply:
Let's keep this as-is.